src/components/Node.vue

```diff
@@ -108,6 +108,9 @@ export default {
     onDisconnected (edge) {
       if (this.edges.includes(edge)) this.edges.splice(this.edges.indexOf(edge), 1)
     },
+    onDelete () {
+      this.$emit('delete', this, this.instance)
```

Comment:
Regarding my comment in Canvas.vue, would just passing `this` not be cleaner here?